website/docs/guide/core-concepts.md

```diff
@@ -0,0 +1,33 @@
+---
+id: core-concepts
+title: Core Concepts
+sidebar_label: Core Concepts
+---
+
+## Fetching data
+
+In general, working with data using lit-pod involves two steps: making a request to some web address to fetch an object containing all data at that address (a [`LitDataset`](../glossary#litdataset)), and then passing that object to functions to extract and manipulate sets of data from it that are relevant to you (which we call [`Thing`s](../glossary#thing)).
+
+<!--
+
+TODO:
+If the spec gets updated to support atomic fetching and updates of multiple Named Graphs at the same Resource,
+this section should be updated to clarify that; the slicing-and-dicing idea might make more sense with that context.
+
+-->
+
+For example, to read my name, you would first fetch a LitDataset from `https://vincentt.inrupt.net/profile/card`. Then, from that LitDataset, you could extract the Thing that represents my profile. From that, you can then read my name.
```

Comment:
```suggestion
For example, to read my name, you would first fetch a LitDataset from `https://vincentt.inrupt.net/profile/card`, the URL of my profile document. Then, from that LitDataset, you could extract the Thing that represents my profile. From that, you can then read my name.
```

Reply:
Hmm, I was kinda avoiding that - does clarifying that that's the URL of my _profile document_ clarify anything here for the reader? (i.e. my hunch is that the reader does not know what a profile document is, so it would only make it _less_ clear? Unless the intent is the other way around, i.e. to use this as an explanation of what a profile document is, but given that that's not the subject of this guide, and that it's not that important of a concept anyway, I'm not sure if that works as intended?)

Reply:
Hmm, I see your point. My intention was to describe why we would be looking for your name at this particular IRI, but not necessarily to describe what a profile document is. Would it add less new concepts if we reused the phrasing from a previous paragraph, e.g. "the URL where my personal information is stored." ?

Reply:
Yes, that makes sense. I'll make the update, thanks.

Reply:
bcf080d6101e8f152820dd452bae057e92ab49ac